setup.py

```diff
@@ -33,7 +33,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.2.2',
+    version='1.2.3-rc.1',
```

Comment:
I thought we are using -rc1. Why is it now -rc.1?

Reply:
Yah, that's what I thought but the "Connector Release Process" and its reference to SemVer indicate this type of version string.